rootston/config.c

```diff
@@ -298,15 +298,18 @@ struct roots_config *parse_args(int argc, char *argv[]) {
 
 	if (result == -1) {
 		wlr_log(L_DEBUG, "No config file found. Using empty config.");
-
-		struct binding_config *bc = calloc(1, sizeof(struct binding_config));
-		wl_list_insert(&config->bindings, &bc->link);
-		bc->command = strdup("exit");
-		bc->modifiers = WLR_MODIFIER_LOGO;
-		bc->keysyms_len = 2;
-		bc->keysyms = calloc(1, bc->keysyms_len * sizeof(xkb_keysym_t));
-		bc->keysyms[0] = XKB_KEY_Meta_L;
-		bc->keysyms[1] = XKB_KEY_q;
+		const char default_config[] =
+			"[bindings]\n"
+			"Logo+Shift+e=exit\n"
+			"Ctrl+q=close\n"
+			"Alt+Tab=next window\n";
```

Comment:
Is it possible to make a function that adds the default bindings programatically that is shared with the parser?